packages/ptokens-helpers/src/validators/index.ts

```diff
@@ -1,19 +1,16 @@
 import { validate } from 'multicoin-address-validator'
-import { NetworkId } from 'ptokens-constants'
+import { ChainId } from 'ptokens-constants'
 
 const validatorFunction =
   (_blockchain: string, _network = 'prod') =>
   (_address: string) =>
     validate(_address, _blockchain, _network)
 
-export const chainIdToAddressValidatorMap: Map<NetworkId, (_address: string) => boolean> = new Map([
-  [NetworkId.ArbitrumMainnet, validatorFunction('eth')],
-  [NetworkId.GnosisMainnet, validatorFunction('eth')],
-  [NetworkId.BscMainnet, validatorFunction('eth')],
-  [NetworkId.PolygonMainnet, validatorFunction('eth')],
+export const chainIdToAddressValidatorMap: Map<ChainId, (_address: string) => boolean> = new Map([
```

Comment:
Why doing a validation based on chain id instead of `BlockchainType`? Definitely less code to maintain in the latter